__tests__/database/schema/UserSchema.test.ts

```diff
@@ -512,7 +514,19 @@ describe('TESTING UserSchema', () => {
           profile_pic_path: 'profilePicture',
           updated_at: 'updatedAt',
           username: 'username',
-          is_banned: false
+          is_banned: false,
+          last_active_at: 'lastActiveAt',
+          user: {
+            banned: false,
+            id: 'userId',
+            name: 'username',
+            username: 'username',
+            last_active_at: 'lastActiveAt',
+            image: 'profilePicture',
+            image:
+              'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
+            created_at: 'createdAt'
+          }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There are two issues here. First, there is a duplicate key `image` in the `user` object which could lead to unexpected behavior. Second, the `image` property has been hardcoded with a URL. It would be better to use a variable or constant instead of hardcoding the URL.
```diff
           profile_pic_path: 'profilePicture',
           updated_at: 'updatedAt',
           username: 'username',
           is_banned: false,
           last_active_at: 'lastActiveAt',
           user: {
             banned: false,
             id: 'userId',
             name: 'username',
             username: 'username',
             last_active_at: 'lastActiveAt',
-            image: 'profilePicture',
-            image:
-              'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
+            image: DEFAULT_PROFILE_PICTURE_URL,
             created_at: 'createdAt'
           }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->